tests/test_date.py

```diff
@@ -489,11 +490,12 @@ def test_get_date_tuple(self, date_string, expected_result):
         self.then_returned_tuple_is(expected_result)
 
     def given_now(self, year, month, day, **time):
+        now = datetime(year, month, day, **time)
         datetime_mock = Mock(wraps=datetime)
-        datetime_mock.utcnow = Mock(return_value=datetime(year, month, day, **time))
-        self.add_patch(
-            patch('dateparser.date_parser.datetime', new=datetime_mock)
-        )
+        datetime_mock.utcnow = Mock(return_value=now)
+        datetime_mock.now = Mock(return_value=now)
```

Comment:
This seems wrong, `datetime.now` should have timezone information.

Reply:
The docs say it shouldn't unless `now` is called with an optional `tz` argument, which seems not to be the case during these tests.

Reply:
I did _not_ know that, interesting!